WBooks/Screens/Library/CustomBookCellModel.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CustomBookCellModel.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 02/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Foundation
```

Comment:
This file was deleted 🤔 maybe it only removes the reference but not the file

Reply:
yep i remove just the reference